### PR TITLE
Update (mainly) zh-* Translations for v1.16

### PR DESCRIPTION
**Pull request by graphemecluster**
Added `zh-HK` translations and addressed some inconsistencies in `zh-TW` and `zh-CN` translations.

Notably:
| en | zh |
|:---:|:---:|
| exit | 離開／离开 |
| quit | 退出 |
| close | 關閉／关闭 |

### CODE REVIEW

File: app/assets/i18n/strings_zh-CN.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
     "online": "在线",
     "open": "打开",
     "queue": "队列",
-    "quickSave": "快速保存",
-    "quickSaveFromFavorites": "快速保存到\"收藏\"",
+    "quickSave": "自动保存",
+    "quickSaveFromFavorites": "自动保存来自“收藏夹”设备的文件",
```
Comment (graphemecluster) [inline app/assets/i18n/strings_zh-CN.json:31]: A number of people misinterpreted it as “quick save *to* favorites” 😄

File: app/assets/i18n/strings_zh-CN.json [hunk 2]
```diff
@@ -255,7 +255,7 @@
     ],
     "author": "作者",
     "contributors": "贡献者",
-    "packagers": "包",
```
Comment (graphemecluster) [inline app/assets/i18n/strings_zh-CN.json:258]: Bun 😆

File: app/assets/i18n/strings_ja.json [hunk 3]
```diff
@@ -28,7 +28,7 @@
     "open": "開く",
     "queue": "順番待ち",
     "quickSave": "クイックセーブ",
-    "quickSaveFromFavorites": "\"Favorites\"からクイックセーブ",
+    "quickSaveFromFavorites": "「お気に入り」からクイックセーブ",
```
Comment (graphemecluster) [inline app/assets/i18n/strings_ja.json:31]: This is only a random walk that checks if people had trouble understanding this line, so there’s no need to credit me as a Japanese translator.

File: app/assets/i18n/strings.json [hunk 4]
```diff
@@ -167,7 +167,7 @@
     },
     "noConnection": {
       "symptom": "Both devices cannot discover each other nor can they share files.",
-      "solution": "Does the problem exist on both sides? If so, you need to make sure that both devices are on the same Wi-Fi network and share the same configuration (port, multicast address, encryption). The Wi-Fi network may not allow communication between participants (AP isolation). In this case, this option must be enabled on the router."
+      "solution": "Does the problem exist on both sides? If so, you need to make sure that both devices are on the same Wi-Fi network and share the same configuration (port, multicast address, encryption). The Wi-Fi network may not allow communication between participants due to Access Point (AP) Isolation. In this case, this option must be disabled on the router."
```
Comment (graphemecluster) [inline app/assets/i18n/strings.json:170]: I guess this must be a mistake

**Comment by Tienisto** [comment]
Thanks!

**Comment by Jesse205** [comment]
In zh-CN, "exit" is usually translated as "退出" instead of "离开".

**Comment by graphemecluster** [comment]
@Jesse205 But then how are “quit” and “exit” distinguished? Not at all?

**Comment by Jesse205** [comment]
Yes,like in macOS,"copy"is translated as"拷贝,"but in Android and Windows,it's"复制."

**Pull request merged**